### PR TITLE
msg sender and value bindings

### LINKED ISSUE #80: Add msg.sender and msg.value bindings
Right now we can't do something like:

```
@notice Stake all my `(self.token(): address).balanceOf(msg.sender(): address)` tokens on #`id`
```
Would that be useful?

### PR DESCRIPTION
**Pull request by fabriziovigevani**
closes #80 

- Passed tx sender and value to evaluator options

- Added check for `msg.sender` and `msg.value` as  a `PropertyAccessExpression`

Example of the syntax: 
```
Send `@tokenAmount(token, msg.value)` from `msg.sender` to `reciever`
```

where 
```
token = address(0)
```

### CODE REVIEW

**Comment by sohkai** [comment]
> @sohkai do you think of a scenario where the msg.data could be useful?

Good catch. Yes, we might as well also add this; in the future we could have helpers or etc. defined that work on the `msg.data`.

**Comment by 0xGabi** [comment]
Great, I let @fabriziovigevani do it and I can release it afterwards

**Comment by fabriziovigevani** [comment]
Yes i will add it in a few minutes :)

File: src/index.js [hunk 1]
```diff
@@ -81,8 +81,9 @@ function evaluate (source, call, { userHelpers = {}, ...options } = {}) {
 
   const availableHelpers = { ...defaultHelpers, ...userHelpers }
 
+  console.log('call tx', call.transaction)
```
Comment (sohkai) [inline src/index.js]: We should remove this :)

**Pull request merged**